.rubocop.yml

```diff
@@ -39,3 +39,8 @@ AllCops:
     - db/schema.rb
     - db/migrate/*
     - vendor/**/*
+
+# Let eval work its magic in the specs
+Lint/Eval:
+  Exclude:
+    - 'spec/**/*'
```

Comment:
Instead of doing this, I'd prefer to write inline comments on the special offending line (we do it in other places :grin:)